neoexchange/core/management/commands/lightcurve_extraction.py

```diff
@@ -536,13 +537,19 @@ def handle(self, *args, **options):
                 if options['title'] is None:
                     sites = ', '.join(mpc_site)
                     try:
-                        if options['timespan'] < 1:
+                        if options['timespan'] < 1 and len(sites) <= 13:
```

Comment:
Document this magic number ?

Reply:
this is just roughly the number of characters before it starts clipping. It's a purely aesthetic choice, really. We could also move to always doing sites on the second line.

Reply:
I was wondering if it could be determined in the code by interrogating the plot dimensions and current font size. Probably not worth the effort if those things rarely change; maybe add a comment similar to the above as to what the '13' is for

Reply:
This feels potentially possible, but requires enough effort that I'd like to have the "what exactly do we want these plots for" conversation first. :) I'll add a comment for now.